tests/test_multiprocess.py

```diff
@@ -129,6 +129,15 @@ def get_pid():
         self.assertEqual(3, self.registry.get_sample_value('c'))
         self.assertEqual(1, c1._value.get())
 
+    def test_multiprocess_registry(self):
```

Comment:
This isn't needed

Reply:
I TDD'd with it ;) is that really worth removing?

Thanks again for all your help! :DDD

Reply:
You don't need a MultiProcessRegistry, so you don't need the test

Reply:
I use MultiProcessRegistry to wrap `MultiProcessCollector`.

How can I gather stats from the directory without it?

Reply:
The user can create a registry themselves, no need for a special one in the client.